### PR TITLE
Update operator-sdk to 1.5.0

### PR DESCRIPTION
**Pull request by oribon**

### CODE REVIEW

**Comment by zshi-redhat** [comment]
@oribon I assume this would be a 4.9 change, right?

**Comment by oribon** [comment]
> @oribon I assume this would be a 4.9 change, right?

@zshi-redhat  Yes, AFAIK

File: controllers/suite_test.go [hunk 1]
```diff
@@ -0,0 +1,83 @@
+/*
```
Comment (mmirecki) [inline controllers/suite_test.go:1]: No tests in this suite

**Pull request merged**